File: Klanvision_backend_node/models/SEOData.js
import { DataTypes } from 'sequelize';
import sequelize from '../config/db.js';

const SEOData = sequelize.define('SEOData', {
  id: {
    type: DataTypes.BIGINT,
    primaryKey: true,
    autoIncrement: true
  },
  siteTitle: {
    type: DataTypes.STRING,
    field: 'site_title'
  },
  metaDescription: {
    type: DataTypes.TEXT,
    field: 'meta_description'
  },
  keywords: {
    type: DataTypes.STRING
  },
  sitemapUrl: {
    type: DataTypes.STRING,
    field: 'sitemap_url'
  },
  googleConsoleId: {
    type: DataTypes.STRING,
    field: 'google_console_id'
  },
  robotsTxt: {
    type: DataTypes.TEXT,
    field: 'robots_txt'
  }
}, {
  tableName: 'seo_data',
  timestamps: false
});

export default SEOData;
export { SEOData };

File: Klanvision_backend/src/main/java/com/klanvision/backend/model/SEOData.java
package com.klanvision.backend.model;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Entity
@Table(name = "seo_data")
@Data
@NoArgsConstructor
@AllArgsConstructor
public class SEOData {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String siteTitle;
    
    @Column(columnDefinition = "TEXT")
    private String metaDescription;
    
    @Column(columnDefinition = "TEXT")
    private String keywords;
    
    private String sitemapUrl;
    private String googleConsoleId;
    
    @Column(columnDefinition = "TEXT")
    private String robotsTxt;
}
